### ISSUE TITLE
Fafnir and Nidhogg spawning simultaneously. 

### ISSUE THREAD

**Issue by Fiocitrine** [DarkstarProject/darkstar#2376]
Nope, can't even. Both are up on oldschool.

**Comment by Hozu** [comment]
Sleep one while you fight the other! Hard mode activate!

**Comment by m241dan** [comment]
It's pretty epic! Get that GM in to despawn one! xD

**Comment by nesstea** [comment]
Double the loot, Double the fun. Only issue I see is QQ.

**Comment by Deadwing888** [comment]
My shell did actually do both, and let me tell you it's really not as much fun as it sounds.

**Comment by Fiocitrine** [comment]
we tried. we only had two RDM, five BLM, and a brd or two. It was cray!

**Comment by teschnei** [comment]
I can spawn 5 or 6 more if you think that will help

**Comment by TeoTwawki** [comment]
only way I can think this would happen is if the server is set to allow both timed and force pop versions at the same time - DeterMob(true) only gets set if the NQ and HQ are using timed pop system, if I recall correctly. So its not so a bug, just an unwelcome feature of the default settings. (perhaps changing the defaults in settings.lua is in order, since this is a rather obscure side effect..)

**Comment by TeoTwawki** [comment]
tl;dr
not entirely unexpected, but if three is a confirmed bug when they are all set to timed pop, I'll fix it since I'm the one that created that setting.

**Comment by Deadwing888** [comment]
Can confirm they double spawn even with no force popping of nq or hq since server reboot.

**Comment by TeoTwawki** [comment]
That doesn't matter - the if force popping is *enabled* at the same time as the timed spawning it will happen, even if they have not been force popped. That they have or have not been force popped is not relevant.

**Comment by TeoTwawki** [comment]
I need to dig into redoing the logic anyway, because I see a potential problem with the kill count forcing HQ and deterring the next NQ while its set to force pop. So its got probs other than what was reported.

**Comment by nesstea** [comment]
Want me to do it @TeoTwawki? That's something I know how to do at least...

**Comment by TeoTwawki** [comment]
na I'll get to it soon

**Comment by nesstea** [comment]
kk

**Comment by TeoTwawki** [comment]
#2376  fixes this

**Comment by TeoTwawki** [comment]
#2403 fixes this

**Comment by TeoTwawki** [comment]
Effing clipboard refusing copy
